coretex/cli/commands/project.py

```diff
@@ -20,47 +20,106 @@
 import click
 
 from ..modules import ui, project_utils, utils, user
-from ...entities import Project
-from ...networking import NetworkRequestError
-from ...configuration import loadConfig, saveConfig
+from ..modules.ui import clickPrompt, errorEcho, successEcho, progressEcho
```

Comment:
Let's use `from modules import ui` and then call these functions with `ui` prefix.